### ISSUE TITLE
[Bug] text contrast ratio of top sites labels is too low

### ISSUE THREAD

**Issue by willfs84** [mozilla-mobile/fenix#13850]
The contrast ratio of the top sites labels is too low especially when you consider how small a font size is being used. Screenshot comparison of Firefox, Chrome and Edge. Notice that both Chrome and Edge have significantly larger font size as well as higher contrast ratio. Look at Firefox vs Edge especially, there's a massive difference there :(  

![topsitestextcontrastratio](https://user-images.githubusercontent.com/47801466/90295256-7766c900-de56-11ea-93a7-7f91458a5cb1.png)

**Comment by willfs84** [comment]
Oops I forgot to add: I'm pretty sure this contrast ratio/text size combination doesn't meet accessibility minimums

**Comment by mcarare** [comment]
Using Accessibility Scanner there are no recommendations for the top sites label. The text is indeed smaller than 12sp, but that was a design decision.

**Comment by liuche** [comment]
hi @asadotzler is there an a11y recommendation here? This wasn't on any of the a11y lists, but let me know if you have a recommendation here (and possibly a priority as well).

**Comment by brampitoyo** [comment]
Sorry. I made a mistake here. I will fix this issue on #15123. Let’s close this one in favour of that.
